README.md

```diff
@@ -1,16 +1,18 @@
-# factory_girl [![Build Status](https://travis-ci.org/thoughtbot/factory_girl.svg)](http://travis-ci.org/thoughtbot/factory_girl?branch=master) [![Dependency Status](https://gemnasium.com/thoughtbot/factory_girl.svg)](https://gemnasium.com/thoughtbot/factory_girl) [![Code Climate](https://codeclimate.com/github/thoughtbot/factory_girl/badges/gpa.svg)](https://codeclimate.com/github/thoughtbot/factory_girl)
+# factory_bot [![Build Status](https://travis-ci.org/thoughtbot/factory_bot.svg)](http://travis-ci.org/thoughtbot/factory_bot?branch=master) [![Dependency Status](https://gemnasium.com/thoughtbot/factory_bot.svg)](https://gemnasium.com/thoughtbot/factory_bot) [![Code Climate](https://codeclimate.com/github/thoughtbot/factory_bot/badges/gpa.svg)](https://codeclimate.com/github/thoughtbot/factory_bot)
 
-factory_girl is a fixtures replacement with a straightforward definition syntax, support for multiple build strategies (saved instances, unsaved instances, attribute hashes, and stubbed objects), and support for multiple factories for the same class (user, admin_user, and so on), including factory inheritance.
+factory_bot is a fixtures replacement with a straightforward definition syntax, support for multiple build strategies (saved instances, unsaved instances, attribute hashes, and stubbed objects), and support for multiple factories for the same class (user, admin_user, and so on), including factory inheritance.
 
-If you want to use factory_girl with Rails, see
-[factory_girl_rails](https://github.com/thoughtbot/factory_girl_rails).
+If you want to use factory_bot with Rails, see
+[factory_bot_rails](https://github.com/thoughtbot/factory_bot_rails).
+
+**A historical note:** factory_bot used to be named factory_girl. An explanation of the name change can be found in the [old factory_girl repository](https://github.com/thoughtbot/factory_girl).
 
 _[Interested in the project name?](NAME.md)._
```

Comment:
I think this line (and the associated file) needs to be removed. Maybe replace with an explanation of the name change, which would also tackle my previous comment?

Reply:
That link now redirects from factory_girl to factory_bot... I'd be interested in knowing why it was renamed? I'm hoping it is something other than "girls shouldn't work in factories."

Reply:
Might be nice to have a link to https://robots.thoughtbot.com/factory_bot in here somewhere.